### ISSUE TITLE
Directory importer creates corrupted disk

### ISSUE THREAD

**Issue by dirkwhoffmann** [dirkwhoffmann/vAmiga#544]
What I did:
- Insert a directory as a disk into df1: 
[WriteKickDisk.zip](https://github.com/dirkwhoffmann/vAmiga/files/7196795/WriteKickDisk.zip)
 
The disk is not usable. Choosing "Export..." runs the analyser. It reports three file system inconsistencies: 
<img width="853" alt="Bildschirmfoto 2021-09-20 um 16 28 14" src="https://user-images.githubusercontent.com/12561945/134029308-49c28cf6-58ec-4e9d-8d10-c832bb087c1f.png">

**Comment by dirkwhoffmann** [comment]
Seems like my FileSystem class doesn't mark the correct bits in the bitmap block when files and directories are added: 

<img width="853" alt="Bildschirmfoto 2021-09-23 um 16 03 23" src="https://user-images.githubusercontent.com/12561945/134533231-606baef5-96e5-4350-b197-350e73c6bade.png">

**Comment by mithrendal** [comment]
But it says nevertheless “all files are good”. 🥸I don’t know what a bitmap block is.

**Comment by dirkwhoffmann** [comment]
"Bitmap blocks contain information about free and allocated blocks. One bit is used per block. If the bit is set, the block is free, a cleared bit means an allocated block."

**Comment by dirkwhoffmann** [comment]
Here is the erroneous bitmap block:

<img width="853" alt="orig" src="https://user-images.githubusercontent.com/12561945/134540932-c97e135d-9d5d-47ba-a5ba-173b6c891ac5.png">

Fortunately, Quarterback (nice tool, BTW) is able to rectify the disk for us... 😎

<img width="853" alt="Bildschirmfoto 2021-09-23 um 16 45 58" src="https://user-images.githubusercontent.com/12561945/134540782-40bcd3ad-d8dc-4259-8950-c0064e7c2e00.png">

Now we know how the bitmap block should look like:

<img width="853" alt="Bildschirmfoto 2021-09-23 um 16 46 57" src="https://user-images.githubusercontent.com/12561945/134541070-aef287bd-9f96-4aa8-a961-6a984ac8c552.png">

**Comment by dirkwhoffmann** [comment]
The bug is most likely fixed. At some point in `FSPartition::locateAllocationBit`, I need to subtract 2 from the provides bit position to skip the two boot blocks (The two boot blocks don't have a dedicated bit in the bitmap. They are implicitly considered to be allocated). Unfortunately, I did the subtraction twice at two different lines in the code.

Now, Quarterback is happy with my disk:

<img width="853" alt="Bildschirmfoto 2021-09-24 um 12 26 08" src="https://user-images.githubusercontent.com/12561945/134762138-bf5a1973-d80a-4f96-b6f7-63357d2613d0.png">

With the creates disk, I was able to run WriteKickDisk successfully inside vAmiga:

<img width="853" alt="Bildschirmfoto 2021-09-24 um 12 36 42" src="https://user-images.githubusercontent.com/12561945/134762153-d3a6d140-7111-4748-bd0a-d52886fb73a4.png">

With the created ADF, it is now possible to create a real Kickstart 1.3 boot disk for the A1000.
